winui/TreeGrid/Export-To-Excel.md

```diff
@@ -0,0 +1,595 @@
+workBook.Worksheets[0].SetRowHeight(2, 50);
+workBook.Worksheets[0].SetColumnWidth(2, 50);
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+## Styling cells based on CellType in Excel
+
+Customize the cell styles based on the `CellType` by using the `GridExportHandler`.
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+options.GridExportHandler = GridExportHandler;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+
+private void GridExportHandler(object sender, TreeGridExcelExportStartOptions e)
+{
+    if (e.CellType == ExportCellType.HeaderCell)
+    {
+        e.Style.Color = Color.Red;
+        e.Style.Font.Color = ExcelKnownColors.White;
+        e.Handled = true;
+    }
+    else if(e.CellType == ExportCellType.RecordCell)
+    {
+        e.Style.Color = Color.LightPink;
+        e.Handled = true;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-To-Excel_images/winui-treegrid-cell-style-customization.png" alt="WinUI TreeGrid displays Customized Cell Style based on CellType in Exported Excel" width="100%" Height="Auto"/>
+
+## Cell customization in Excel while exporting
+
+Customize the cells by setting the `CellsExportHandler` in the `TreeGridExcelExportOptions`.
+
+### Customize cell value while exporting
+
+Customize the cell values while exporting to excel by using the `CellsExportHandler` in the `TreeGridExcelExportOptions`.
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+options.CellsExportHandler = CellsExportHandler;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+
+private void CellsExportHandler(object sender, TreeGridCellExcelExportOptions e)
+{
+    // Based on the column mapping name and the cell type, we can change the cell values while exporting to excel.
+    if (e.CellType == ExportCellType.RecordCell && e.ColumnName == "Availability")
+    {
+        if (e.CellValue.Equals(true))
+            e.Range.Cells[0].Value = "Y";
+
+        else
+            e.Range.Cells[0].Value = "N";
+        e.Handled = true;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-To-Excel_images/winui-treegrid-export-customization.png" alt="Customizing Cell Values while Exporting to Excel in WinUI TreeGrid" width="100%" Height="Auto"/>
+
+Here, the cell values changed for the Availability column are based on the custom condition.
+
+### Changing row style in Excel based on data
+
+You can customize the rows based on the record values by using the `CellsExportHandler`.

```

Comment:
Customize the rows based on the record values by using the `CellsExportHandler.`

Reply:
Updated